drivers/axi_core/spi_engine/spi_engine.c

```diff
@@ -140,6 +140,26 @@ static int32_t spi_engine_set_transfer_width(struct spi_desc *desc,
 	return SUCCESS;
 }
 
+/**
+ * @brief Set SPI engine clock frequency
+ *
+ * @param desc Decriptor containing SPI Engine's parameters
+ * @param speed_hz SPI engine transfer speed
+ * @return uint32_t This function always returns SUCCESS
```

Comment:
General question: Shouldn't we set function that always return true to return void?